tests/scale_test/notifier.py

```diff
@@ -138,13 +138,21 @@ def cli():
     help='The path to the folder to save metrics of this notifier.',
     show_default=True,
 )
+@click.option(
+    '--force/--no-force',
+    default=False,
+    help='Whether or not to force analysis when a workflow with functionally '
+    'equivalent metadata already exists',
```

Comment:
"True to re-analyze data even though its functional equivalent exists already." ... maybe?